index.js

```diff
@@ -29,8 +30,10 @@ module.exports = (projectPath, opts) => {
   const getDepInfo = deps =>
     Object.keys(deps || {}).map(dep => {
       const depPath = path.join(projectPath, 'node_modules', dep);
+      // Catch handles dependencies that are not installed
```

Comment:
Hehe, spent an hour trying to figure out why everything was failing silently. Promises are neat, but can be a real pain in the ass to debug and figure out where in the chain it's failing.

I was debating adding `catch` on every promise but figured that's better left for after this PR is in